lib/commands/owner.js

```diff
@@ -83,13 +85,40 @@ class Owner extends BaseCommand {
     }
   }
 
+  async execWorkspaces ([action, ...args], filters) {
+    await this.setWorkspaces(filters)
+    // ls pkg or owner add/rm package
+    if ((action === 'ls' && args.length > 0) || args.length > 1) {
+      const implicitWorkspaces = this.npm.config.get('workspace', 'default')
+      if (implicitWorkspaces.length === 0) {
+        log.warn(`Ignoring specified workspace(s)`)
+      }
+      return this.exec([action, ...args])
+    }
+    for (const [name] of this.workspaces) {
+      switch (action) {
+        case 'ls':
+        case 'list':
+          return this.ls(name)
+        case 'add':
+          return this.changeOwners(args[0], name, 'add')
+        case 'rm':
+        case 'remove':
+          return this.changeOwners(args[0], name, 'rm')
+        default:
+          throw this.usageError()
+      }
+    }
+  }
+
   async ls (pkg) {
-    pkg = await this.getPkg(pkg)
+    pkg = await this.getPkg(this.npm.prefix, pkg)
     const spec = npa(pkg)
 
     try {
       const packumentOpts = { ...this.npm.flatOptions, fullMetadata: true, preferOnline: true }
-      const { maintainers } = await pacote.packument(spec, packumentOpts)
+      const packument = await pacote.packument(spec, packumentOpts)
+      const { maintainers } = packument
```

Comment:
`packument` doesn't seem otherwise used, was this just a temporary change for debugging?